src/pages/create-identity/AuthorizedSigner/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AnimatePresence } from "motion/react";
+import { FormProvider, useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+
+import Topbar from "@/components/Topbar";
+import NavigateBack from "@/components/NavigateBack";
+import StepIndicator from "@/components/StepIndicator";
+import routes from "@/constants/routes";
+
+import RequiredInformation from "./RequiredInformation";
+import EmailVerification from "./EmailVerification";
+import PreviewIdentity from "./PreviewIdentity";
+
+const formSchema = z.object({
+  name: z.string().min(1),
+  email: z.string().email(),
+});
+
+export default function AuthorizedSigner() {
+  const [currentStep, setCurrentStep] = useState(0);
+  const navigate = useNavigate();
+
+  const methods = useForm({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      name: "",
+      email: "",
+    },
+  });
+
+  const steps = [
+    <RequiredInformation
+      continueToNextStep={() => {
+        setCurrentStep((current) => current + 1);
+      }}
+    />,
+    <EmailVerification
+      continueToNextStep={() => {
+        setCurrentStep((current) => current + 1);
+      }}
+    />,
+    <PreviewIdentity
+      continueToNextStep={() => {
+        navigate(`${routes.CREATE_IDENTITY}/${routes.SUCCESS}`);
+      }}
+      editRequiredInformation={() => {
+        setCurrentStep((current) => current - 1);
```

Comment:
```suggestion
        setCurrentStep(0);
```

Need to go back to the first page.